src/pymor/reductors/mt.py

```diff
@@ -0,0 +1,210 @@
+# This file is part of the pyMOR project (http://www.pymor.org).
+# Copyright 2013-2020 pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (http://opensource.org/licenses/BSD-2-Clause)
+
+import numpy as np
+import scipy.linalg as spla
+from pymor.algorithms.gram_schmidt import gram_schmidt, gram_schmidt_biorth
+from pymor.algorithms.samdp import samdp
+from pymor.algorithms.to_matrix import to_matrix
+from pymor.core.base import BasicObject
+from pymor.models.iosys import LTIModel, sparse_min_size
+from pymor.operators.constructions import IdentityOperator
+from pymor.operators.numpy import NumpyMatrixOperator
+from pymor.parameters.base import Mu
+from pymor.reductors.basic import LTIPGReductor
+
+
+class MTReductor(BasicObject):
+    """Modal Truncation reductor.
+
+    See Section 9.2 in [A05]_.
+
+    Parameters
+    ----------
+    fom
+        The full-order |LTIModel| to reduce.
+    mu
+        |Parameter values|.
+    """
+
+    def __init__(self, fom, mu=None):
+        assert isinstance(fom, LTIModel)
+        if not isinstance(mu, Mu):
+            mu = fom.parameters.parse(mu)
+        assert fom.parameters.assert_compatible(mu)
+        self.fom = fom
+        self.mu = mu
+        self.V = None
+        self.W = None
+        self._pg_reductor = None
+
+    def reduce(self, r=None, decomposition='samdp', projection='orth',
+               symmetric=False, which="NR", method_options=None, take_complex=False):
+        """Modal Truncation.
+
+        Parameters
+        ----------
+        r
+            Order of the reduced model.
+        decomposition
+            Algortihm to use for the decomposition:
+
+            - `'eig'`: use scipy.linalg.eig algorithm
+            - `'samdp'`: find dominant poles using samdp algorithm
+        projection
+            Projection method:
+
+            - `'orth'`: projection matrices are orthogonalized with
+              respect to the Euclidean inner product
+            - `'biorth'`: projection matrices are biorthogolized with
+              respect to the E product
+        symmetric
+            `True` if Operator A is symmetric and E is symmetric, positive
```

Comment:
I rarely, if ever, see a comma here.

Also, for boolean arguments, it's common in pyMOR to document them with "If `True`, ..." (e.g., see `gram_schmidt`). Here it could be "If `True`, assume A is symmetric and E is symmetric positive definite".